code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -635,3 +636,139 @@
 
 /obj/item/borg/upgrade/modkit/tracer/adjustable/attack_self(mob/user)
 	bolt_color = input(user,"","Choose Color",bolt_color) as color|null
+
+//Megafauna modkits
+
+//Blood Drunk
+/obj/item/borg/upgrade/modkit/sharpnel
+	name = "sharpnel shot modification kit"
+	desc = "Your shots will now make targets bleed."
+	denied_type = /obj/item/borg/upgrade/modkit/sharpnel //So it wont stack
+	cost = 35
+	modifier = 1
+
+/obj/item/borg/upgrade/modkit/sharpnel/projectile_prehit(obj/item/projectile/kinetic/K, atom/target, obj/item/gun/energy/kinetic_accelerator/KA)
+	if(isliving(target))
+		var/mob/living/L = target
+		var/datum/status_effect/stacking/saw_bleed/B = L.has_status_effect(STATUS_EFFECT_SAWBLEED)
+		if(!B)
+			L.apply_status_effect(STATUS_EFFECT_SAWBLEED, modifier)
+		else
+			B.add_stacks(modifier)
+
+//Bubblegum
+/obj/item/borg/upgrade/modkit/shotgun
+	name = "shotgun blast modification kit"
+	desc = "Makes you fire 3 kinetic shots instead of one."
```

Comment:
why

Reply:
ok u gotta agree a shotgun ka would be fun

but. 120 brute at pb range.

Reply:
120 fucking brute at pb range? Literally one hit crit. Bruh moment.